demo/index.html

```diff
@@ -6,13 +6,14 @@
 
 		<title>cosmoz-image-viewer demo</title>
 
-		<script src="../../webcomponentsjs/webcomponents-lite.js"></script>
+		<script src="../../webcomponentsjs/webcomponents-lite.js"></script>		
```

Comment:
Trailing spaces.. should be auto-fixed with vscode?
Missing vscode settings?

Reply:
Worked only in a javascript block.

Added now `"files.trimTrailingWhitespace": true`